ApplicationLibCode/GeoMech/GeoMechVisualization/RivGeoMechVizLogic.h

```diff
@@ -43,13 +43,13 @@ class RivGeoMechVizLogic : public cvf::Object
     ~RivGeoMechVizLogic() override;
 
     void appendNoAnimPartsToModel( cvf::ModelBasicList* model );
-    void appendPartsToModel( int timeStepIndex, cvf::ModelBasicList* model );
-    void updateCellResultColor( int timeStepIndex, RimGeoMechCellColors* cellResultColors );
-    void updateStaticCellColors( int timeStepIndex );
+    void appendPartsToModel( int viewerStepIndex, cvf::ModelBasicList* model );
```

Comment:
The updated naming of variables here improves understanding and readability.

Clearly three concepts: `viewerStepIndex`, `timeStepIndex` and `frameIndex`